frontend/workflows/projectCatalog/src/details/info/types.tsx

```diff
@@ -0,0 +1,42 @@
+import type { CHIP_VARIANTS } from "@clutch-sh/core";
+import type { IconProp } from "@fortawesome/fontawesome-svg-core";
+
+export interface ProjectInfoChip {
+  text: string;
+  title?: string;
+  icon?: any;
+  url?: string;
+  variant?: typeof CHIP_VARIANTS[number];
+}
+
+export interface ProjectMessenger {
+  text: string;
+  icon?: IconProp;
+  url?: string;
+}
+
+interface ProjectRequests {
+  number: number;
+  url?: string;
+  type: string;
+}
+
+export interface ProjectRepository {
+  name: string;
+  repo: string;
+  url?: string;
+  icon?: IconProp;
+  requests?: ProjectRequests;
+}
+
+export interface ProjectInfo {
```

Comment:
Should this project info come from the ProjectAPI definitions?

Reply:
Yeah that makes sense. This was a hold over from when I went super generic. I've removed the types files and utilize the ProjectAPI typings now.